docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -22,7 +22,7 @@ Galaxy is a web-based platform for data analysis. Galaxy is open source and can
 Using Existing Galaxy Servers
 -----------------------------
 
-Institutions across the world maintain Galaxy instances that are shared with the greater community and are a great way to learn and use Galaxy. Some of these instances are subsidized by grants and offered for free (such as the `Galaxy US <https://usegalaxy.org/>`__, `Galaxy EU <https://usegalaxy.eu/>`__, and `Galaxy AU <https://usegalaxy.org.au/>`__) and some are available for a fee (such as `Galaxy AnVIL <https://anvil.terra.bio/>`__ or `Galaxy Pro <https://researcher.galaxyworks.io/>`__). A lot of great learning resources are available on the the `Galaxy Training Network <https://training.galaxyproject.org/>`__, with help on specific questions available via `Galaxy Help <https://help.galaxyproject.org/>`__.
+Institutions across the world maintain Galaxy instances that are shared with the greater community and are a great way to learn and use Galaxy. Some of these instances are subsidized by grants and offered for free (such as the `Galaxy US <https://usegalaxy.org/>`__, `Galaxy EU <https://usegalaxy.eu/>`__, and `Galaxy AU <https://usegalaxy.org.au/>`__) and some are available for a fee (such as `Galaxy AnVIL <https://anvil.terra.bio/>`__ or `Galaxy Pro <https://galaxyworks.io/pricing>`__). A lot of great learning resources are available on the the `Galaxy Training Network <https://training.galaxyproject.org/>`__, with help on specific questions available via `Galaxy Help <https://help.galaxyproject.org/>`__.
```

Comment:
Avani did some digging and found that Galaxyworks "decided to shut down that service," so we should just remove that link entirely, ie https://github.com/dockstore/dockstore-documentation/pull/159/commits/bbad3c7f736ed86db88615e5b3e4fb100602b6de

Reply:
Good catch, thanks!